octopusdeploy/resource_account.go

```diff
@@ -26,6 +26,7 @@ func resourceAccount() *schema.Resource {
 				},
 				Optional: true,
 			},
+			// Azure
```

Comment:
`account_type` applies to all accounts, it's the thing that determines whether it's Azure/AWS/UsernamePasword, etc

Reply:
I removed those as they are no longer needed.